express/blocks/sticky-footer/sticky-footer.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { createTag, fetchPlaceholders, readBlockConfig } from '../../scripts/scripts.js';
+
+/**
+ * @param {number} [rating=5]
+ */
+function createStars(rating = 5) {
+  if (typeof rating === 'string') {
+    // eslint-disable-next-line no-param-reassign
+    rating = Number.parseFloat(rating);
+  }
+  const rHalves = Math.round(rating / 0.5);
+  const whole = Math.floor(rHalves / 2);
+  let hasHalf = rHalves % 2;
+
+  let str = '';
+  for (let i = 0; i < 5; i += 1) {
+    let type = 'star44';
+    if (i < whole) {
+      type = 'star44';
+    } else if (hasHalf) {
+      type = 'star-half44';
+      hasHalf = 0;
+    } else {
+      type = 'star-empty44';
+    }
+
+    str += `<svg xmlns="http://www.w3.org/2000/svg" class="icon icon-star">
+      <use href="/express/icons/ccx-sheet_44.svg#${type}"></use>
+    </svg>`;
```

Comment:
```suggestion
    str += getIcon(type);
```
(import from `scripts.js`)